doc/dev/autograph.rst

```diff
@@ -852,3 +852,34 @@ rather than runtime.
 
 For more details, see the :ref:`compile-time vs. runtime <compile_time>`
 documentation.
+
+
+Adding modules for Autograph conversion
+---------------------------------------
+
+Due to the experimental state of Autograph support, the following modules 
```

Comment:
This is not due to the experimental state of the feature, rather it is a necessity because library code is not meant to be converted (and very often cannot be converted). What is meant to be converted is application code. TensorFlow does the same by excluding itself and other known libraries from conversion.

Reply:
Done